src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -46,6 +46,17 @@ public MetadataProvider(Docset docset, Cache cache)
             return _metadataCache.GetOrAdd(file, GetMetadataCore);
         }
 
+        public static JsonSchema LoadMetadataSchema(Config config, string docsetPath, string fallbackDocsetPath)
```

Comment:
Why not make metadataSchema as a property of MetadataProvider and expose it to TemplateEngine